test/testdata/go-header-template_good.go

```diff
@@ -0,0 +1,6 @@
+/*MY TITLE.*/
+
+//golangcitest:args -Egoheader
+//golangcitest:config_path testdata/configs/go-header-template.yml
```

Comment:
Just a question – how it works without `test/` prefix?
Looks like your `filepath.Join` is not covered 🤔